tsconfig.json

```diff
@@ -5,7 +5,13 @@
     "target": "es2015",
     "outDir": "lib",
     "lib": ["es2018"],
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noFallthroughCasesInSwitch": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true
```

Comment:
`skipLibCheck` avoids typechecking `.d.ts` files in dependencies. With our strict options some modules of coc.nvim would not typecheck, but we don't really care.